crest/Assets/Crest/Crest/Scripts/Shapes/OceanWaveSpectrum.cs

```diff
@@ -136,6 +146,9 @@ public float GetAmplitude(float wavelength, float componentsPerOctave, out float
             power = hasNextIndex ? Mathf.Lerp(thisPower, nextPower, alpha) : thisPower;
             power = Mathf.Pow(10f, power);
 
+            var c = ComputeWaveSpeed(wavelength);
+            power *= Mathf.Clamp01(Mathf.InverseLerp(windSpeed, windSpeed * 0.95f, c));
```

Comment:
I'm finding that even at maximum wind speed, this line will produce some zeros. They are for wavelengths 1152 and 2304. Just checking that this is expected. I am guessing they are too big even for wind speed 150.

Reply:
Yeah its a struggle with the wind speed slider. We need an exponential slider otherwise its impossible to tweak the low speeds. Any ideas how we might achieve this? I suppose brute force is to make a new editor for the ocean component, then implement the nonlinear range bespokely. but let me know if you have fancier ideas around property drawers or etc.

in any case i propose to address this in a follow up and ill merge this as is.

Reply:
A custom property drawer using the system we have should be achievable. I can look into this.